.github/workflows/deploy-lambda-phacToSbc.yml

```diff
@@ -3,11 +3,12 @@ name: Deploy Lambda Function
 on:
   push:
     tags:
-      - lambda-prod
+      - phactosbc-lambda-prod
```

Comment:
Should both Lambdas be deployed at the same time? Considering a code change to one (in the shared layer) affects both, we may prefer to always keep both in sync.

Reply:
Every time a new lambda is deployed a new layer version is also created and this new lambda will point to it. The other lambda still points to the old layer version and can only be updated if we run the tag command accordingly (or change the layer manually in the AWS console). This could avoid some regression tests every time on both functions when dealing with only one. What do you think?